landing_page_backend/contacts/models.py

```diff
@@ -18,4 +18,4 @@ class Contact(models.Model):
     email = models.EmailField()
     number = models.IntegerField()
     message = models.TextField()
-    time_create = models.DateTimeField(auto_now_add=True)
+    create = models.DateTimeField(auto_now_add=True)
```

Comment:
Should be named 'created'. Field should not be named as verb. Functions name still should start with verb.